xtask/src/ci.rs

```diff
@@ -140,6 +144,16 @@ fn run_feature_tests(cmd: Option<FeatureSet>) -> Result<()> {
     Ok(())
 }
 
+fn run_crypto_features() -> Result<()> {
```

Comment:
"run features" doesn't make a lot of sense to me, can we rename to `_tests` or `_checks`?